src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -79,20 +80,25 @@ object Boot extends IOApp with LazyLogging {
     }
   }
 
-  private[sam] def createLdapConnectionPool(directoryConfig: DirectoryConfig): cats.effect.Resource[IO, LDAPConnectionPool] = {
-    val dirURI = new URI(directoryConfig.directoryUrl)
+  private[sam] def createLdapConnectionPool(directoryUrl: String, user: String, password: String, connectionPoolSize: Int, name: String): cats.effect.Resource[IO, LDAPConnectionPool] = {
```

Comment:
Why expanding it to individual parameters?

Reply:
so it can be called with different pool size, once for the foreground pool, once for the background pool